app/src/main/java/org/mozilla/fenix/settings/TabsSettingsFragment.kt

```diff
@@ -58,6 +59,10 @@ class TabsSettingsFragment : PreferenceFragmentCompat() {
         // pref_key_tab_view_grid and look into using the native RadioGroup in the future.
         listRadioButton = requirePreference(R.string.pref_key_tab_view_list_do_not_use)
         gridRadioButton = requirePreference(R.string.pref_key_tab_view_grid)
+        searchTermTabGroups = requirePreference<SwitchPreference>(R.string.pref_key_search_term_tab_groups).also {
```

Comment:
@MozillaNoah We have to make sure we only show this preference when the feature flag is turned on. Did you verify this? I think this needs a `it.visible = FeatureFlags.tabSearchGroup` (not sure what the name is)

Reply:
See also: https://github.com/mozilla-mobile/fenix/blob/main/app/src/main/java/org/mozilla/fenix/settings/TabsSettingsFragment.kt#L80

Reply:
Thanks for catching this. I did it for the inactive tabs toggle but not here. Will update shortly.